spec/models/article_spec.rb

```diff
@@ -0,0 +1,19 @@
+require "rails_helper"
+
+RSpec.describe Article, type: :model do
+  describe "valid factory" do
+    subject { build(:article) }
+
+    it { is_expected.to be_valid }
+  end
+
+  describe "validations" do
```

Comment:
may be need subject

Reply:
Already updated: before inside this describe.